api/batch/resource_test.go

```diff
@@ -119,8 +123,12 @@ var (
 )
 
 func TestCreateSparkApplicationResource(t *testing.T) {
-	models.InitKubernetesLabeller("gojek.com/") //nolint:errcheck
-	defer models.InitKubernetesLabeller("")     //nolint:errcheck
+	err := models.InitKubernetesLabeller("gojek.com/", environmentName)
+	assert.NoError(t, err)
+
+	defer func() {
+		_ = models.InitKubernetesLabeller("", "")
+	}()
```

Comment:
Did this to make the error checking explicit (also stops any IDE from pointing out that the error is unhandled).